### ISSUE TITLE
Arithmetic error in xpbar-active-tracking.lua

### ISSUE THREAD

**Issue by CosmicCleric** [TheLinuxITGuy/Turtle-Dragonflight#97]
(Turtle WoW client version 1.17.2 (7205)(Release) Nov 20 2024.)

Line 19 of in xpbar-active-tracking.lua was reporting an arithmetic error because of a nil value. Unfortunately the error scrolled off so I can't recite it verbatim, but I am certain of the lua file and line number, as I had opened the file manually while the error was still displayable. 

It seems to happen when my rested xp gets to zero and I enter a place that has rested xp gaining (inn, etc.) and I mouse over the xp bar. I have "Always show detailed XP" and "Continuous rest XP" options in the 'XP & Reputation' options area checked/on/true.

I was able to change line 19 to make the error go away. My revision ...

`if IsResting() and ((GetXPExhaustion("Player") == 0) or (GetXPExhaustion("player") / UnitXPMax("player") ~= 1.5))  then`

Basically I just swapped the "==0" check with the division being done, on the 'or' statement. My _guess_ is that `GetXPExhaustion("player")` can return a nil instead of 0 in rare circumstances (like being in an inn, etc.).

**Comment by TheLinuxITGuy** [comment]
Fixed in 2.8.1
